zkevm-circuits/src/blake2b/circuit/src/lib.rs

```diff
@@ -0,0 +1,1855 @@
+/*  The basic building blocks of this circuit are called GATES. In this context a gate is a map between tuples of values, 
```

Comment:
This file might be too large (i.e. 2k+ LOC). Consider splitting into modules

Reply:
Perhaps, it can be splitted, but it should be done without harm to convenience of documentation understanding.

Reply:
True, this is optional tho